ddsketch/mapping/cubically_interpolated_mapping.go

```diff
@@ -75,6 +75,10 @@ func (m *CubicallyInterpolatedMapping) Value(index int) float64 {
 	return m.approximateInverseLog((float64(index)-m.normalizedIndexOffset)/m.multiplier) * (1 + m.relativeAccuracy)
```

Comment:
Assuming that `lowerBound` is inlined (or that there is no effect on performance), can we update it?
```suggestion
	return m.LowerBound(index) * (1 + m.relativeAccuracy)
```